### PR TITLE
[inner-rce] add readOnly prop + fix text color

### PR DESCRIPTION
**Pull request by noamtwix**

### CODE REVIEW

File: packages/editor/web/src/RichContentEditor/InnerRCE.jsx [hunk 1]
```diff
@@ -46,8 +46,12 @@ class InnerRCE extends Component {
     e.stopPropagation();
   };
 
+  focus = () => this.ref.focus();
+
+  setRef = ref => (this.ref = ref);
+
   render() {
-    const { theme, isMobile, ...rest } = this.props;
+    const { theme, isMobile, additionalProps, editing = true, ...rest } = this.props;
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/InnerRCE.jsx:54]:
```suggestion
    const { theme, isMobile, additionalProps, readOnly, ...rest } = this.props;
```

File: packages/editor/web/src/RichContentEditor/RichContentEditor.jsx [hunk 2]
```diff
@@ -532,11 +533,12 @@ class RichContentEditor extends Component {
         onBlur={onBlur}
         onFocus={onFocus}
         textAlignment={textAlignment}
+        readOnly={readOnly || false}
       />
     );
   };
 
-  renderInnerRCE = (contentState, callback, renderedIn, setFocusToBlock) => {
+  renderInnerRCE = ({ contentState, callback, renderedIn, additionalProps }) => {
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/RichContentEditor.jsx:541]: what's this?

Comment (noamtwix) [reply packages/editor/web/src/RichContentEditor/RichContentEditor.jsx:541]: props where the plugins can send to the inner rce and the inner rce pass to <RichContentEditor />
e.g. placeholder, onBackspace...

Comment (noamtwix) [reply packages/editor/web/src/RichContentEditor/RichContentEditor.jsx:541]: this change was allready in Aviv & Dana branches 
I just want to add it to master

Comment (noamtwix) [reply packages/editor/web/src/RichContentEditor/RichContentEditor.jsx:541]: removing setFocusToBlock is cleanup that should have been done earlier and not related to the adding of additionalProps

Comment (shaulgo) [reply packages/editor/web/src/RichContentEditor/RichContentEditor.jsx:541]: I think they should probably be specific. But you can merge it like this for now.

**Review by shaulgo — APPROVED** [review]

**Pull request merged**